collectors/prometheus.go

```diff
@@ -67,6 +67,9 @@ type MonitoringConfig struct {
 
 	// DisableGraph disables collection of graph metrics
 	DisableGraph bool
+
+	// DisableHtlc disables collection of HTLCs metrics
+	DisableHtlc bool
```

Comment:
This commit needs to come before the other one, otherwise the first one won't compile on its own.